annotations/forms.py

```diff
@@ -122,3 +122,19 @@ class Meta:
         model = VogonUser
         fields = ('full_name', 'email', 'affiliation', 'location', 'link', )
 
+    def clean(self):
```

Comment:
``clean()`` is a good place to put validation that involves field interdependencies, but since these are checking fields independently we should add a separate ``clean_[fieldname]`` method for each. See [Form and field validation](https://docs.djangoproject.com/en/1.9/ref/forms/validation/).